packages/labs/cli/package.json

```diff
@@ -3,17 +3,20 @@
   "description": "Tooling for Lit development",
   "version": "0.0.0",
   "author": "Google LLC",
+  "license": "BSD-3-Clause",
   "bin": {
     "lit": "./bin/lit.js"
   },
   "bugs": "https://github.com/lit/lit/issues",
   "type": "module",
   "main": "lib/index.js",
+  "types": "lib/index.d.ts",
```

Comment:
Do we want types here given that it's not importable?

Reply:
Done